inspect_camoufox_options.py

```
@@ -0,0 +1,6 @@
+import camoufox
+print(f"Launch options: {dir(camoufox.launch_options)}")
+try:
+    help(camoufox.Camoufox)
+except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)